generators/generator-bot-adaptive/generators/app/templates/assets/schemas/update-schema.ps1

```diff
@@ -8,7 +8,7 @@ Write-Host "Running schema merge."
 if (Test-Path $SCHEMA_FILE -PathType leaf) { Move-Item -Force -Path $SCHEMA_FILE -Destination $BACKUP_SCHEMA_FILE }
 if (Test-Path $UISCHEMA_FILE -PathType leaf) { Move-Item -Force -Path $UISCHEMA_FILE -Destination $BACKUP_UISCHEMA_FILE }
 
-bf dialog:merge "*.schema" "!sdk-backup.schema" "*.uischema" "!sdk-backup.uischema" "!sdk.override.uischema" "../*.csproj" -o $SCHEMA_FILE
+bf dialog:merge "*.schema" "!**/sdk-backup.schema" "*.uischema" "!**/sdk-backup.uischema" "!**/sdk.override.uischema" "../*.csproj" -o $SCHEMA_FILE
```

Comment:
Do we not need to include ../package.json to accommodate for JS bots?

Reply:
Adding "../package.json" does work.  So one script could work for both.  Can you think of any negative side effects to that?  Any reason a package.json would exist in a DotNet project?